spec/system/planning_applications/recommending_spec.rb

```diff
@@ -30,7 +30,11 @@
     context "with no previous recommendations" do
       it "can create a new recommendation, edit it, and submit it" do
         click_link "In assessment"
-        click_link planning_application.reference
+
+        within("#under_assessment") do
```

Comment:
Because Capybara can 'see' the content of all the tabs, it doesn't know whether to click the planning application reference in the 'In assessment' tab or the 'All applications' tab.